macros/generate_insert_statements_post_hook.sql

```diff
@@ -0,0 +1,21 @@
+{% macro generate_insert_statements_post_hook(relation, resource_type='nodes', relationships=False, batch_size=var('insert_batch_size', 500 if target.type == 'bigquery' else 10000)) %}
```

Comment:
Was there a reason for adding the `_post_hook` suffix to this macro name?

Reply:
1. no! can remove the suffix if we don't like it
2. I was getting jinja rendering issues when trying to add this config in `dbt_project.yml` -- i believe it was because I was trying to have the outputs be ints, and you have to wrap jinja-in-yml in double quotes, so it was interpreting the setting as a string. I mostly just punted on this, may have been doing something wrong

Reply:
1. I say remove the suffix, just to shorten the macro name... also technically it's calling the insert statements not just generating, so I'd maybe go for a name that's a bit clearer
2. Hmm not sure I follow - we have other project level variables that are numeric (ex: `max_depth_dag`)

Reply:
1. can do!
2. I was attempting to add the variable as "{{ 500 if target.type == 'bigquery' else 10000 }}"` and I think the fact that the numeric value gets wrapped in the double quotes (which are necessary for the jinja to render) it ends up being interpreted as a string

Reply:
settled for `insert_resources_from_graph`